examples/00-basic/12-get_material_properties.py

```diff
@@ -0,0 +1,60 @@
+"""
+.. _ref_get_material_properties:
+
+Get material properties from the result file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Material properties are assigned to each element in APDL and by default they
+are written out in the APDL result file. We can extract material properties
+of each element using PyDPF by getting the properties from the `meshed_region`.
```

Comment:
As much as possible, try to link to existing class or method documentation. One easy way to figure out correct class name is to search within the [PyDPF-Core Documentation](https://dpf.docs.pyansys.com/) for the class name. For example, [MeshedRegion](https://dpf.docs.pyansys.com/api/ansys.dpf.core.meshed_region.html).

I've provided the suggestion below.

```suggestion
of each element using PyDPF by getting the properties from the :class:`meshed_region <ansys.dpf.core.meshed_region.MeshedRegion>`.
```